### PR TITLE
feat: Add get sticker pack method

### PR DESCRIPTION
**Pull request by Kondrahin**
# Description

Add get sticker pack method.

# Checklist

- [x] Linters and tests have passed
- [ ] I've written a changelog for PR change
- [ ] I'll make a release when PR is approved

### CODE REVIEW

File: botx/client/stickers_api/get_sticker_pack.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+from botx.models.api_base import UnverifiedPayloadBaseModel, VerifiedPayloadBaseModel
+from botx.models.stickers import Sticker, StickerPack
+
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal  # type: ignore  # noqa: WPS440
+
+
+def response_exception_thrower(
```
Comment (Nimond) [inline botx/client/stickers_api/get_sticker_pack.py:19]: Import it from `botx/client/botx_method.py`

File: botx/client/stickers_api/get_sticker_pack.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    status: Literal["ok"]
+    result: BotXAPIGetStickerPackResult
+
+    def to_domain(self) -> StickerPack:
+        return StickerPack(
+            id=self.result.id,
+            name=self.result.name,
+            is_public=self.result.public,
+            stickers_order=self.result.stickers_order,
+            stickers=[
```
Comment (Nimond) [inline botx/client/stickers_api/get_sticker_pack.py:64]: Sort using `stickers_order` and remove `stickers_order` from domain model

File: botx/client/stickers_api/get_sticker_pack.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        payload: BotXAPIGetStickerPackRequestPayload,
+    ) -> BotXAPIGetStickerPackResponsePayload:
+        jsonable_dict = payload.jsonable_dict()
+        sticker_pack_id = jsonable_dict.pop("sticker_pack_id")
+        path = f"/api/v3/botx/stickers/packs/{sticker_pack_id}"
+
+        response = await self._botx_method_call(
+            "GET",
+            self._build_url(path),
+            json=jsonable_dict,
```
Comment (Nimond) [inline botx/client/stickers_api/get_sticker_pack.py:88]: U dont need json here

File: botx/client/stickers_api/get_sticker_pack.py [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+        404: response_exception_thrower(StickerPackNotFoundError),
+    }
+
+    async def execute(
+        self,
+        payload: BotXAPIGetStickerPackRequestPayload,
+    ) -> BotXAPIGetStickerPackResponsePayload:
+        jsonable_dict = payload.jsonable_dict()
+        sticker_pack_id = jsonable_dict.pop("sticker_pack_id")
+        path = f"/api/v3/botx/stickers/packs/{sticker_pack_id}"
```
Comment (Nimond) [inline botx/client/stickers_api/get_sticker_pack.py:83]: `path = f"/api/v3/botx/stickers/packs/{jsonable_dict['sticker_pack_id']}"`

**Review by Nimond — APPROVED** [review]

**Review by lxmnk — APPROVED** [review]

**Pull request merged**